index.html

```diff
@@ -8,340 +8,305 @@
   <link rel="stylesheet" href="style.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
 </head>
-  <header>
-    <a href="#"><h2>John Doe</h2></a>
-    <nav>
-      <button type="button" id="toggleBtn"><img src="/images/menu.png" class="menu" alt=""></button>
-      <ul class="menu_items" id="menuList">
-        <li><button type="button" id="btn_close">X</button></li>
-        <li><a href="#project-s">Portfolio</a><span>></span></li>
-        <li><a href="#headline">About</a><span>></span></li>
-        <li><a href="#contact-me">Contact</a><span>></span></li>
-        <li><img src="/images/mail.png" class="mail" alt="image"><span>></span></li>
+<body>
+    <section id="detail1">
+      <button type="button" id="modal-close">X</button>
+      <div id="detail"></div>
+      <!-- <section class="title">
+        <h2>Multi-Post stroies</h2>
+      </section>
+  
+      <figure><img src="/images/portfolio.png" alt="Project Image"></figure>
+      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+        when an unknown printer took a galley of type and scrambled it 1960s with the 
+        releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, 
+        when an unknown printer took a galley of type veris lapoa todoe.</p>
+      
+      <ul class="tags">
+        <li>html</li>
+        <li>Ruby on rails</li>
+        <li>CSS</li>
       </ul>
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻